### DISCUSSION TITLE
Engine output up and down repeat problem

### DISCUSSION THREAD

**Discussion by sujyun** [flybywiresim/aircraft#2786] [Help]
There is a problem with the engine output.
Outputs between 200 and 290 knots repeat rises and falls, sometimes without any problems, but this often occurs during flight.

Now it's connected between Simbrief and 320NX, so it's convenient to use. Is this related? 

https://user-images.githubusercontent.com/76735752/103286664-0e0dde00-4a24-11eb-941c-7157998b035b.mp4

**Comment by robinottawa** [comment]
Is this new? Is it the normal changes when reaching altitude, etc?
I just watched, so, no. You are level. But it's strange to be at this low speed at this altitude, no? 
Just trying to brainstorm. No idea.

**Comment by MapleNAV-TCASFAULT** [comment]
I believe this issue has been fixed in the latest dev version
